js/lib/jquery.transloadit.js

```diff
@@ -118,10 +118,12 @@ const tus = require('tus-js-client')
       this._isOnline = true
 
       this._error = null
+      this.$ = $
```

Comment:
Perhaps we can fallback to window.$ if it’s not set for BC reasons?

Reply:
Why would it not be set? Is there any way to use this class outside this file?

Reply:
You're right, sorry. The project's author is away for a while and i thought I'd step in but am not super familiar with the code yet 👌